annotation-server/src/gene-phenotypes/gene-phenotypes.service.ts

```diff
@@ -14,6 +13,9 @@ import { Phenotype } from './entities/phenotype.entity';
 export class GenePhenotypesService {
     private readonly logger = new Logger(GenePhenotypesService.name);
 
+    private hashedGeneSymbols: Map<string, GeneSymbol>;
+    private hashedPhenotypes: Map<string, Phenotype>;
```

Comment:
Could we use `CacheMap`s here?

Reply:
Using `CacheMaps` would be a bit much here since they

1. are abstract meaning we would have to implement another class just for this and
2. offer error handling that we don't need